exercises/concept/community-garden/test/community_garden_test.exs

```diff
@@ -1,40 +1,40 @@
 defmodule CommunityGardenTest do
   use ExUnit.Case
 
+  @task_id 1
   test "start returns an alive pid" do
     assert {:ok, pid} = CommunityGarden.start()
     assert Process.alive?(pid)
   end
 
+  @task_id 2
   test "when started, the registry is empty" do
     assert {:ok, pid} = CommunityGarden.start()
     assert [] == CommunityGarden.list_registrations(pid)
   end
 
+  @task_id 3
   test "can register a new plot" do
     assert {:ok, pid} = CommunityGarden.start()
     assert %Plot{} = CommunityGarden.register(pid, "Johnny Appleseed")
   end
 
+  @task_id 3
   test "maintains a registry of plots" do
     assert {:ok, pid} = CommunityGarden.start()
     assert %Plot{} = plot = CommunityGarden.register(pid, "Johnny Appleseed")
     assert [plot] == CommunityGarden.list_registrations(pid)
   end
 
+  @task_id 4
   test "can release a plot" do
     assert {:ok, pid} = CommunityGarden.start()
     assert %Plot{} = plot = CommunityGarden.register(pid, "Johnny Appleseed")
     assert :ok = CommunityGarden.release(pid, plot.plot_id)
     assert [] == CommunityGarden.list_registrations(pid)
   end
 
-  test "can get a specific registration by id" do
```

Comment:
I don't see a difference between this test and the next test. I think it might be an accident that there are two so similar tests?